packages/grid/package.json

```diff
@@ -21,11 +21,16 @@
   "sideEffects": false,
   "types": "dist/typings/index.d.ts",
   "dependencies": {
+    "@popperjs/core": "^2.4.4",
     "@zendeskgarden/container-splitter": "^0.2.3",
     "@zendeskgarden/container-utilities": "^0.7.1",
+    "@zendeskgarden/react-buttons": "^8.52.0",
+    "@zendeskgarden/react-tooltips": "^8.52.0",
+    "@zendeskgarden/svg-icons": "6.32.0",
     "polished": "^4.0.0",
     "prop-types": "^15.5.7",
-    "react-merge-refs": "^1.1.0"
+    "react-merge-refs": "^1.1.0",
+    "react-popper": "^2.2.3"
```

Comment:
Builds fail when adding tooltip without these libraries.

Reply:
Ah right, of course. Might highlight the opportunity to change tooltip placement based on the placement of the splitter button. But this can probably be a future conversation with design.

Reply:
We should be using `"react-popper": "^1x"` to stay consistent with `react-tooltips`. It would remove the need for `@popperjs/core`. Technically `@zendeskgarden/react-tooltips` should be bringing in `react-popper` as a dependency.

Reply:
Fixed per https://github.com/zendeskgarden/react-components/pull/1371#discussion_r891332366